source/_components/alarm_control_panel.manual.markdown

```diff
@@ -205,3 +205,53 @@ automation:
       - service: alarm_control_panel.alarm_disarm
         entity_id: alarm_control_panel.house_alarm
 ```
+
+Sending a Notification when the Alarm is Armed (Away/Home), Disarmed and in Pending Status
+
+```- alias: 'Send notification when alarm is Disarmed'
```

Comment:
See https://developers.home-assistant.io/docs/en/documentation_standards.html#templates

All examples containing Jinja2 templates should be wrapped outside of the code markdown with the `{% raw %}` tag.